src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
```

Comment:
Think about which CSS properties in this component need to be customizable, and express them as CSS variables.

Reply:
its already been used in the file with the class `.clickable`

Reply:
ah sorry i understood what you meant now, to make use of the --var. will update the code

Reply:
✔